modules/ROOT/pages/otel-support.adoc

```diff
@@ -0,0 +1,145 @@
+
+== Before You Begin
+
+To use the OpenTelemetry Mule features, you must be familiar with:
+
+* Distributed tracing concepts, including traces and spans. If you're new to distributed tracing and OpenTelemetry, we recommend starting with https://opentelemetry.io/docs/concepts/what-is-opentelemetry/[What is OpenTelemetry] and https://opentelemetry.io/docs/concepts/observability-primer/#understanding-distributed-tracing[Understanding distributed tracing].
+* xref:monitoring::telemetry-exporter.adoc[How to export telemetry data to third-party monitoring systems].
+
+Before starting with the observability standard, ensure that you have:
+
+* For flow tracing, at least Mule runtime 4.6.0.
+* For distributed tracing, HTTP connector 1.8, and the span exporter configured.
+
+== Tracing Mule App
+
+Mule runtime leverages OpenTelemetry to provide:
+
+* App execution observability 
++
+This includes how each component of the app behaves. The main span involved in a Mule app is the flow span. Each Mule component execution taking part in the message processing is represented as a child span of the corresponding flow.
+* Distributed tracing
++
+This enables you to track Mule app interactions. When multiple systems and services are involved, distributed tracing tracks app requests as they flow through distributed environments providing a comprehensive view of the app's execution.
+
+For example, if an HTTP Listener receives https://www.w3.org/TR/trace-context/[W3C Trace Context] headers, the flow span in Mule acquires the client's remote parent span, the span where the HTTP call originated. Mule automatically propagates the current span to the server endpoints when using the HTTP Requester component. 
+
+The span context format supported for receiving and propagating the context is the https://www.w3.org/TR/trace-context/[W3C Trace Context] format.
+
+=== Mule Span Data
+
+Traces are a sequence of spans representing a complete operation in a distributed system. Spans can be nested and represent a unit of work or operation. Each span includes: 
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Categories | Description | Possible Values
+| Name | Name of the span | “mule:set-variable”
+| Parent Span ID | Span ID of current span's parent | “86838830d494d679”
+| Start and End Timestamps | Earliest and latest timestamp for a span | "2021-10-22 16:04:01.209458162 +0000 UTC"
+| Span Context | Contains data that uniquely identifies the span: trace ID, span ID, trace flags, trace state | 
+| Attributes | Key-value pairs containing metadata used to annotate a span to carry information about the operation it's tracking. | {
+“location”: “flow/processors/2”,
+“correlation.id”:  “1234abcd”
+}
+| Span Events | A structured log message or annotation on a span, used to denote a meaningful, singular point in time during the span's duration. | {
```

Comment:
```suggestion
| Span Events | A structured log message or annotation on a span, used to denote a meaningful, singular point in time during the span's duration | {
```

Reply:
Oh, ignore the period removal. I see that you have a full sentence later in the table.